### PR TITLE
CIF-1748 - Content Fragment Commerce component

### PR DESCRIPTION
**Pull request by LSantha**
* added commerce content fragment proxy component
 * added UI tests for commerce content fragment component edit dialog (see [content-fragment-component-dialog.js ](https://github.com/adobe/aem-cif-guides-venia/pull/138/files#diff-11098fbdaf13958df011f78c1ffdcb18485e67816967e9cd36f928fea28ab6f6) as the relevant change)
 * updated UI test dependencies to the level of the UI test dependencies of CIF add-on
 * added prettier to the project and reformatted the JS files accoridingly

## Related Issue

CIF-1748

## How Has This Been Tested?

UI tests, manually.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.tests/test-module/.eslintrc.js [hunk 1]
```diff
@@ -13,14 +10,11 @@ module.exports = {
         ecmaVersion: 9
     },
     rules: {
-        'semi': ['error'],
+        semi: ['error'],
```
Comment (mhaack) [inline ui.tests/test-module/.eslintrc.js:13]: Why are single quotes removed here for some of the rule entries? Should be either all or non? How is that related to your change?

**Review by mhaack — COMMENTED** [review]
This has some many format changes, I guess done by prettier? What is the actual change to review here? Only the Venia proxy component?

File: ui.tests/test-module/.eslintrc.js (hunk 1 above)
Comment (LSantha) [reply ui.tests/test-module/.eslintrc.js:13]: That's also the result of the prettier reformatting. I suppose the quotes are needed for the properties with '-' in their name.

Comment (LSantha) [reply ui.tests/test-module/.eslintrc.js:13]: As I wrote in the PR description the relevant new JS file containing the tests is content-fragment-component-dialog.js .

**Review by mhaack — APPROVED** [review]

**Review by herzog31 — APPROVED** [review]
